arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -5049,6 +5073,24 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		007A4255256C7F480025F582 /* Add execution permission to masquerade script */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Add execution permission to masquerade script";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# The project uses a bash program named *masquerade* in a custom build rule to conceal app secrets in versioned source code. Grant masquerade executable permissions.\nchmod a+rwx,g-wx,o-wx \"${SRCROOT}/Scripts/masquerade\"\n";
+		};
```

Comment:
This build phase seems unnecessary as everything seems to work without it. Even it is necessary for the script to be executable, why not check in the script with those permissions instead of setting them each build?

Reply:
Good point. I removed it from the project. 
How to setup "masquerade" should not be instructed by the sample viewer; we can refer to other tutorials or docs. More to discuss about it next week.